SteamBot/SteamTradeDemoHandler.cs

```diff
@@ -202,9 +202,11 @@ public override void OnTradeSuccess()
         public override void OnTradeAwaitingEmailConfirmation(long tradeOfferID)
         {
             Log.Warn("Trade ended awaiting email confirmation");
-            SendChatMessage("Please complete the email confirmation to finish the trade");
+            SendChatMessage("Please complete the email confirmation to finish the trade, you've got 10 minutes from now.");
```

Comment:
This mechanism is also usable for regular trade offers which can be cancelled.

Ideally, you shouldn't stop polling. However, current implementation adds individual polling for each trade offer which creates lots of overhead for active bots.